### DISCUSSION TITLE
I'm trying to load a font for DrawString().

### DISCUSSION THREAD

**Discussion by winbamstudios** [CosmosOS/Cosmos#2393] [Q&A]
The problem is i have to use a Cosmos.Graphics.Font. Not any other kind of font.

**Comment by zarlo** [comment]
???

**Reply by winbamstudios** [reply]
using a canvas.drawstring function. i need to load in a ttf font like roboto or smth the font libraries ive used dont work

**Reply by zarlo** [reply]
you have to use a PCScreenFont and i think that uses Bitmap Distribution Format

**Reply by winbamstudios** [reply]
ok
